packages/cosmos_ui_components/lib/cosmos_text_theme.dart

```diff
@@ -17,6 +17,19 @@ abstract class CosmosTextTheme {
         height: 1.27,
       );
 
+  TextStyle get titleS => const TextStyle(
+        fontFamily: "Inter",
+        fontWeight: FontWeight.w700,
+        fontSize: 13,
+        height: 1.27,
```

Comment:
as far as I can see in the designs, the line height is `15.99` which divided by font size gives: `15.99/13 = 1.23`